src/daemon/daemon.cpp

```diff
@@ -2996,7 +3019,7 @@ mp::Daemon::async_wait_for_ready_all(grpc::ServerReaderWriterInterface<Reply, Re
                                      const std::vector<std::string>& vms, const std::chrono::seconds& timeout,
                                      std::promise<grpc::Status>* status_promise, const std::string& start_errors)
 {
-    fmt::memory_buffer errors;
+    fmt::memory_buffer errors, warnings;
     fmt::format_to(std::back_inserter(errors), "{}", start_errors);
```

Comment:
I know this also stems from the original code. But maybe move the `errors` and `warnings` variables closer to the code blocks that use them, so we can localize the variables better. 

 `fmt::memory_buffer warnings;` can be moved to the line before the code block. 
```
{
std::lock_guard<decltype(start_mutex)> lock{start_mutex};
```
and 
```
    fmt::memory_buffer errors;
    fmt::format_to(std::back_inserter(errors), "{}", start_errors);
```
can be moved to the line before 
`    for (const auto& future : start_synchronizer.futures())
`

Reply:
Right, looks better this way. Changed.